app/components/viral/alert_component.rb

```diff
@@ -14,10 +14,11 @@ class AlertComponent < Component
       danger: 'danger'
     }.freeze
 
-    def initialize(type: TYPE_DEFAULT, message: nil)
+    def initialize(type: TYPE_DEFAULT, message: nil, **system_arguments)
       @type = TYPE_MAPPINGS[type.to_sym] || TYPE_DEFAULT
       @message = message
       @classes = classes_for_alert
+      @system_arguments = system_arguments
```

Comment:
**suggestion:** We can now update this to include all the classes and role for the alert containing `div`.

```ruby
def initialize(type: TYPE_DEFAULT, message: nil, **system_arguments)
      @type = TYPE_MAPPINGS[type.to_sym] || TYPE_DEFAULT
      @message = message
      @system_arguments = system_arguments

      classes = classes_for_alert
      @system_arguments[:classes] =
        class_names('flex p-4 border-l-4', classes, @system_arguments[:classes])
      @system_arguments[:role] = 'alert'
    end
```

Reply:
Please see c34dff8f39491f77cc589c6e8fd116e2f7b79e36.
I noticed you removed classes `mb-4 text-sm`. If you don't like the small text can we keep the bottom margin to leave some space b/w the alert and the samples header/title?

Reply:
I think I would rather it's wrapping container handle the surrounding layout for it.  You can always pass in the `mb-4`  into `classes` if you want

Reply:
Please see 4c63c285190756cd14a743bfb25ddffe671ac60e.

Reply:
Thanks